### DISCUSSION TITLE
Page load complete event

### DISCUSSION THREAD

**Discussion by tingfeng-key** [php-webdriver/php-webdriver#908] [Q&A] [answered]
Hello, I want to do something after the page is fully loaded, but the browser will be closed before the page is loaded. For example, I want to stay ten seconds after the page is loaded.Is there any way to solve my problem? thank you

**Comment by andrewnicols** [comment, marked as answer]
You will need to use the execute function to execute client-side JS to check the page ready state and wait until your condition is met.
